src/app/stack/card-details/component-information/component-information.component.html

```diff
@@ -42,28 +87,35 @@
                         </div> -->
                         <!-- Security Card -->
 
-                        
-                        
+
+
                         <!-- Insights Card -->
                         <div *ngSwitchCase="'alternate'">
                             <!-- alternate -->
                             <span [innerText]="comp?.recommendation?.componentInformation?.name || '------'"></span>
-                            <span class="recommendation-reason" [tooltip]="comp?.recommendation?.reason" *ngIf="comp?.recommendation?.reason && comp?.recommendation?.componentInformation?.name">Why this dependency?</span>
+                            <span class="recommendation-reason" [tooltip]="comp?.recommendation?.reason"
+                                *ngIf="comp?.recommendation?.reason && comp?.recommendation?.componentInformation?.name">Why
+                                this dependency?</span>
                         </div>
                         <div *ngSwitchCase="'feedback'">
                             <!-- feedback -->
-                            <component-feedback [genericInformation]="genericInformation" [feedback]="component?.recommendation?.feedback || component?.feedback"></component-feedback>
+                            <component-feedback [genericInformation]="genericInformation"
+                                [feedback]="component?.recommendation?.feedback || component?.feedback">
+                            </component-feedback>
                         </div>
                         <div *ngSwitchCase="'confidence'">
                             <!-- confidence -->
-                            <ana-progress-meter *ngIf="comp?.recommendation?.confidenceScore" [config]="comp?.recommendation?.confidenceScore"></ana-progress-meter>
-                            <ana-progress-meter *ngIf="component?.confidenceScore" [config]="component?.confidenceScore"></ana-progress-meter>
-                            <span *ngIf="!comp?.recommendation?.confidenceScore && !component?.confidenceScore">-----</span>
+                            <ana-progress-meter *ngIf="comp?.recommendation?.confidenceScore"
```

Comment:
Here you need to use cooccurrence_probability for confidenceScore

Reply:
fixed